app/controllers/assessor_interface/assessment_sections_controller.rb

```diff
@@ -18,22 +23,8 @@ def update
 
     private
 
-    def load_assessment_section_and_assessment
-      @assessment_section =
-        AssessmentSection
-          .includes(assessment: :application_form)
-          .where(
-            assessment_id: params[:assessment_id],
-            assessment: {
-              application_form_id: params[:application_form_id]
-            }
-          )
-          .find_by!(key: params[:key])
-
-      @assessment = @assessment_section.assessment
-      @application_form = @assessment.application_form
-      @qualifications = @application_form.qualifications.ordered
-      @work_histories = @application_form.work_histories.ordered
+    def assessment_section_view
+      @assessment_section_view ||= AssessmentSectionViewObject.new(params:)
```

Comment:
```suggestion
    def assessment_section_view_object
      @assessment_section_view_object ||= AssessmentSectionViewObject.new(params:)
```

I'd suggest this name to avoid conflicts with "views", but don't feel strongly on this.